pkg/kudoctl/util/repo/repo.go

```diff
@@ -57,6 +59,20 @@ func (r Repositories) CurrentRepo() *RepositoryConfiguration {
 	return r.GetRepo(r.Context)
 }
 
+// RepositoryConfig gets the repo configuration from settings
+func RepositoryConfig(fs afero.Fs, settings *env.Settings) (*RepositoryConfiguration, error) {
+	r, err := LoadRepositories(fs, settings.Home.RepositoryFile())
+	if err != nil {
+		// this allow for no client init... perhaps we should return the error requesting kudo init
+		r = NewRepoFile()
```

Comment:
hmm so this looks like it's getting some kind of file but I had to actually go to the implementation to find out it's not.  So should it be `GetDefaultConfig`?

also I don't understand what is `Context` inside that configration, since we just assign some default repo name to it?

Reply:
also please se something better than `r`, with `r` and `repo` in the same context it's just a bit confusing what is what

Reply:
yeah... we have a repositories file, with a list of repo configs which allow us to create a operator repo.   I'll clean this up.

Reply:
many of the structs "stuttered" such as `repo.OperatorRepository`... that is really a `repo.Client`.   Refactoring these will go a long way to make it easy to reason about.